src/components/styles/PageSection.styled.ts

```diff
@@ -12,23 +12,42 @@ const fadeInUp = keyframes`
     }
 `
 
-export const MainContainer = styled.div<{ imgPosition?: 'left' | 'right' }>`
-  display: grid;
+export const MainContainer = styled.div<{ imgPosition?: 'left' | 'right'; responsiveReverse?: boolean }>`
+  display: flex;
   justify-content: space-between;
-  grid-template-rows: minmax(0, 1fr);
-  grid-template-columns: ${(props) => props.imgPosition && 'auto'} auto;
-  grid-template-areas: '${(props) => props.imgPosition === 'left' && 'image'} text ${(props) =>
-    props.imgPosition === 'right' && 'image'}';
-  gap: 5rem;
   animation-duration: 1s;
   animation-name: ${fadeInUp};
+
+  @media screen and (min-width: 700px) {
+    display: grid;
+    grid-template-rows: minmax(0, 1fr);
+    grid-template-columns: ${(props) => props.imgPosition && 'auto'} auto;
+    grid-template-areas: '${(props) => props.imgPosition === 'left' && 'image'} text ${(props) =>
+      props.imgPosition === 'right' && 'image'}';
+    gap: 5rem;
+  }
+
+  @media screen and (max-width: 700px) {
+    flex-direction: ${(props) => (props.responsiveReverse ? 'column-reverse' : 'column')};
+    align-items: flex-start;
+    padding: 0px;
```

Comment:
this can do without units! also, might want to add gap if the sections are too close?

Reply:
Resolved in ea80b2e